nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/coordinator/WriteCoordinator.scala

```diff
@@ -205,7 +206,22 @@ class WriteCoordinator(commitLogCoordinator: Option[ActorRef],
         Future(RecordRejected(db, namespace, metric, bit, List(s"no data actor for node ${location.node}")))
     }
 
-  override def receive: Receive = {
+  override def receive: Receive = warmUp
+
+  def warmUp: Receive = {
```

Comment:
maybe a couple of lines of documentation is needed just to quickly explain the meaning of the two behaviours